dev-python/hiredis/Manifest

```diff
@@ -1,2 +1 @@
 DIST hiredis-0.2.0.tar.gz 46113 BLAKE2B bb0479f99f5f705488846ff5e7b52d6e8a5c34519936aa747027e8a597271d3ba70efc2455f54c24248091a4477ac2a307cad7d89a03c96cb7d331e6b6e82a30 SHA512 9eb18ec089517340043d14938a213b139dc948dc9cf2f9c65e632bb4694ac22543b87a5875c1cea083e2bc8e303103c749047889cdb3e9d5680b586d34330c86
-DIST hiredis-1.0.1.tar.gz 54460 BLAKE2B 90bdbd99f47b0765636dbeee9fc3cbb9c71fd7ba752a0d58ab9c2c9d2ee93bfc1a343465618d1127703ae09a4870d6691d7d5b52f59150849a448eae73c6a1d3 SHA512 8aa5e29b09f42826d746857d55b8042b40ffa3977386b0f8b2ea1b2ede0c28337f3df4bbd177192d8873570810bc9d0987f1f2b29f094f27adaa4aaf409c3a6b
```

Comment:
But... why drop the latest ebuild instead of 0.2.0?

Reply:
Oops.